test/chunk.ts

```diff
@@ -0,0 +1,75 @@
+import { makeChunk, makeSpan, Utils } from '@fairdatasociety/bmt-js'
```

Comment:
let's introduce the convention that the test filenames should be identical to the smart contract's filenames